src/components/MediaList/MediaList.tsx

```diff
@@ -19,17 +17,30 @@ import {
   metaTypes,
 } from "@ndla/licenses";
 import type { MetaType } from "@ndla/licenses";
+import { Heading, Text } from "@ndla/primitives";
 import { SafeLink } from "@ndla/safelink";
-import { Text } from "@ndla/typography";
+import { styled } from "@ndla/styled-system/jsx";
 import { LicenseLink } from "@ndla/ui";
 import LicenseBylineDescriptionList from "./LicenseBylineDescriptionList";
-
-export const MediaList = styled.ul`
-  padding-left: 0;
-  display: flex;
-  flex-direction: column;
-  margin: ${spacing.normal} 0;
-`;
+import FavoriteButton from "../../components/Article/FavoritesButton";
+
+export const MediaList = styled("ul", {
+  base: {
+    listStyle: "none",
+    padding: "0",
+    width: "100%",
+    "& h3,p ": {
+      marginBottom: "3xsmall",
+    },
+    tabletDown: {
+      "& button, a": {
+        width: "100%",
+      },
+    },
+  },
+});
+
+export const StyledFavoriteButton = styled(FavoriteButton, { base: { width: "auto!" } });
```

Comment:
Hvorfor trengs dette? Og la oss heller kalle denne `MediaListFavoriteButton`

Reply:
Det var for å fikse at hjerteknappen tar for mye plass på mobil